.golangci.yml

```diff
@@ -0,0 +1,66 @@
+run:
+  timeout: 5m
+
+linters:
+  enable:
+    - bidichk
+    - bodyclose
+    - dogsled
+    - dupl
+    - errcheck
+    - errorlint
+    - exportloopref
+    - gocheckcompilerdirectives
+    - goconst
+    - gocritic
+    - gofmt
+    - gofumpt
+    - goimports
+    - revive
+    - gosec
+    - gosimple
+    - govet
+    - ineffassign
+    - lll
+    - loggercheck
+    - misspell
+    - nakedret
+    - nilerr
+    - nilnil
+    - nolintlint
+    - prealloc
+    - protogetter
+    - exportloopref
+    - staticcheck
+    - stylecheck
+    - typecheck
+    - unconvert
+    - unused
+    - wastedassign
+    - whitespace
+    - nolintlint
+    - asciicheck
+    - dupword
+
+issues:
+  exclude-rules:
+    - path: _test\.go
+      linters:
+        - gosec
+        - revive
+    - linters:
+        - lll
+      source: "https://"
+  max-same-issues: 50
+
```

Comment:
Based on https://olegk.dev/go-linters-configuration-the-right-version (shameless plug):

```
output:
  # Must have. Easier to understand the output.
  print-linter-name: true

  # No, no skips, everything should be reported.
  uniq-by-line: false

  # To be honest no idea when this can be needed, maybe a multi-module setup?
  path-prefix: ""

  # Slightly easier to follow the results + getting deterministic output.
  sort-results: true
```